build.sbt

```diff
@@ -151,10 +151,6 @@ lazy val commonSettings = Seq(
 
 def scalacCrossOptions(scalaVersion: String) =
```

Comment:
Could strip out the entire function for clarity, or leave it for future convenience.

Reply:
I think it's worth leaving this in and just removing the 2.11 case. I suspect we'll want to add support for 2.13 at some point and it wouldn't surprise me if scalc options changed again.

Reply:
I was leaning that way too :+1: